ivadomed/config/config.json

```diff
@@ -74,19 +74,22 @@
         "metadata": "contrasts",
         "film_layers": [0, 1, 0, 0, 0, 0, 0, 0, 0, 0]
     },
-    "testing_parameters": {
-        "binarize_prediction": -1,
-        "uncertainty": {
-            "epistemic": false,
-            "aleatoric": false,
-            "n_it": 0
-        }
+    "uncertainty": {
+        "epistemic": false,
+        "aleatoric": false,
+        "n_it": 0
+    },
+    "postprocessing": {
+        "remove_noise": {"thr": -1},
+        "keep_largest": {},
+        "binarize_prediction": {"thr": 0.5},
+        "fill_holes": {},
+        "remove_small": {"unit": "vox", "thr": 3},
+        "uncertainty": {"thr": -1, "suffix": "_unc-vox.nii.gz"}
```

Comment:
```suggestion
        "binarize_prediction": {"thr": 0.5},
        "uncertainty": {"thr": -1, "suffix": "_unc-vox.nii.gz"},
        "fill_holes": {},
        "remove_small": {"unit": "vox", "thr": 3}
```

Reply:
I am suggesting this order and the unc thresholding could somewhat induce some holes etc.